docs/deployment/file-access-auth.md

```diff
@@ -0,0 +1,159 @@
+| `CertificateSha256` | `Processes`  | String     | No       | None       | SHA256 of the leaf certificate of the instigating process. |
+| `CDHash`            | `Processes`  | String     | No       | None       | CDHash of the instigating process. |
+
+
+### Example Configuration
+
+This is an example configuration conforming to the specification outlined above:
+
+```plist
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Version</key>
+	<string>v0.1-experimental</string>
+	<key>WatchItems</key>
+	<dict>
+		<key>UserFoo</key>
+		<dict>
+			<key>Paths</key>
+			<array>
+				<string>/Users/*/foo</string>
+				<dict>
+					<key>Path</key>
+					<string>/Users/*/tmp/foo</string>
+					<key>IsPrefix</key>
+					<true/>
+				</dict>
+			</array>
+			<key>Options</key>
+			<dict>
+				<key>AllowReadAccess</key>
+				<false/>
+				<key>AuditOnly</key>
+				<true/>
+			</dict>
+			<key>Processes</key>
+			<array>
+				<dict>
+					<key>BinaryPath</key>
+					<string>/usr/local/bin/my_foo_writer</string>
+					<key>TeamID</key>
+					<string>ABCDEF1234</string>
+				</dict>
+			</array>
+		</dict>
+	</dict>
+</dict>
+</plist>
+```
+
+## Details
+
+### Rule Matching
+
+When an operation occurs on a path that matches multiple configured path globs, the rule containing the "most specific" matching path glob is applied (i.e., the longest matching path).
+
+For example, consider the configured rules and paths:
+```
+RULE_1: /tmp/foo     [IsPrefix=true]
+RULE_2: /tmp/foo.txt [IsPrefix=false]
+RULE_3: /tmp         [IsPrefix=true]
+```
+The following table demonstrates which rule will be applied for operations on a given path:
+| Operation Path   | Rule Applied | Reason |
+| ---------------- | ------------ | ------ |
+| /tmp/foo         | `RULE_1`     | Matches prefix, more specific than `RULE_3` |
+| /tmp/foo/bar     | `RULE_1`     | Matches prefix, more specific than `RULE_3` |
+| /tmp/bar         | `RULE_3`     | Matches prefix |
+| /tmp/foo.txt     | `RULE_2`     | Matches literal, more specific than `RULE_1` |
+| /tmp/foo.txt.tmp | `RULE_1`     | Matches prefix, more specific than `RULE_3`, literal match doesn't apply |
+| /foo             | N/A          | No rules match operations on this path |
+
+If a configuration contains multiple rules with duplicate configured paths, only one rule will be applied to the path. It is undefined which configured rule will be used. Administrators should take care not to define configurations that may have duplicate paths.
+
+### Path Globs
+
+Configured path globs represent a point in time. That is - path globs are expanded when a configuration is applied to generate the set of monitored paths. Currently this is not a "live" representation of the filesystem. For instance, if a new file or directory is added that would match a glob after the configuration is applied, it is not immediately monitored.
```

Comment:
```suggestion
Configured path globs represent a point in time. That is, path globs are expanded when a configuration is applied to generate the set of monitored paths. Currently, this is not a "live" representation of the filesystem. For instance, if a new file or directory is added that would match a glob after the configuration is applied, it is not immediately monitored.
```

Reply:
Optional: Remove "Currently" --> "This not a 'live' representation..."

I would avoid using "Currently" since it implies (to me) that at some point a "live" representation of the filesystem would be supported. While that might be true, it's possible to frustrate users with expectations that aren't meant according to what that timeframe could look like to them.